package.json

```
@@ -28,50 +28,51 @@
     "babel-plugin-transform-es2015-modules-commonjs": "6.26.2",
     "babel-plugin-transform-object-rest-spread": "6.26.0",
     "babel-preset-react": "6.24.1",
-    "chai": "4.1.2",
+    "chai": "4.2.0",
     "chai-json-schema": "1.5.0",
     "co-task": "1.0.0",
     "cpx": "1.5.0",
-    "enzyme": "3.3.0",
-    "enzyme-adapter-react-16": "1.1.1",
+    "enzyme": "3.7.0",
+    "enzyme-adapter-react-16": "1.6.0",
     "eslint": "4.19.1",
     "eslint-plugin-import": "2.11.0",
-    "eslint-plugin-json": "1.2.0",
+    "eslint-plugin-json": "1.2.1",
     "eslint-plugin-mozilla": "0.16.0",
     "eslint-plugin-no-unsanitized": "3.0.0",
     "eslint-plugin-promise": "3.7.0",
     "eslint-plugin-react": "7.7.0",
     "eslint-watch": "3.1.4",
     "husky": "0.14.3",
-    "istanbul-instrumenter-loader": "0.2.0",
+    "istanbul-instrumenter-loader": "3.0.1",
     "joi-browser": "13.0.1",
-    "karma": "2.0.2",
+    "karma": "3.1.1",
     "karma-chai": "0.1.0",
-    "karma-coverage": "1.1.1",
+    "karma-coverage-istanbul-reporter": "2.0.4",
     "karma-firefox-launcher": "1.1.0",
     "karma-mocha": "1.3.0",
     "karma-mocha-reporter": "2.2.5",
     "karma-sinon": "1.0.5",
     "karma-sourcemap-loader": "0.3.7",
-    "karma-webpack": "3.0.0",
+    "karma-webpack": "3.0.5",
     "loader-utils": "0.2.16",
     "minimist": "1.2.0",
     "mocha": "5.1.1",
     "mock-raf": "1.0.0",
     "node-fetch": "2.1.2",
-    "node-sass": "4.7.2",
+    "node-sass": "^4.9.4",
```

I assume this really wants to be 

```suggestion
    "node-sass": "4.9.4",
```

---

Good catch!